bins/revme/src/statetest/models/deserializer.rs

```diff
@@ -31,7 +32,7 @@ where
     let output = if let Some(stripped) = string.strip_prefix("0x") {
         U256::from_str_radix(stripped, 16).unwrap()
     } else {
-        U256::from_dec_str(&string).unwrap()
+        U256::from_str(&string).unwrap()
     };
```

Comment:
`U256` implements std's `FromStr`with support for dec, bin, oct, hex. So this whole statement can be replaced with

```
let output = string.parser().unwrap();
```


See: 

https://github.com/recmo/uint/blob/baf8006583bce32d7c906756ee93454a6be6013b/src/string.rs#L167

Reply:
The same applies to `deserialize_str_as_u64` btw.